lib/badge-data.js

```diff
@@ -69,9 +86,19 @@ function makeLabel(defaultLabel, overrides) {
   return overrides.label || defaultLabel;
 }
 
+function getSimpleIcon(icon = '', color = 'whitesmoke'){
+  icon = typeof icon === 'string' ? icon.toLowerCase() : '';
+  if (!simpleIcons[icon]){
+    return undefined;
+  }
+  color = makeColor(color) || 'whitesmoke';
+  const svg = simpleIcons[icon].svg.replace('<svg', `<svg fill="${color}"`);
+  return 'data:image/svg+xml;base64,' + Buffer.from(svg).toString('base64');
+}
+
 function makeLogo(defaultNamedLogo, overrides) {
   if (overrides.logo === undefined){
-    return logos[defaultNamedLogo];
+    return logos[defaultNamedLogo] || getSimpleIcon(defaultNamedLogo, overrides.logoColor);
```

Comment:
For parity, is it possible to support `logoColor` in our own logos?

Reply:
Should be able to, Do you know if there is a reason that we load the logos into memory on launch?
_(in `load-logos.js`)_
As that is the main reason i haven't tried yet, because it loads them and keeps them as base64.

Reply:
Seems to be okay :+1:

Reply:
It’s a performance optimization, to load and encode them once instead of per request. We could cache the unencoded logos instead, or keep both in memory, as a small optimization for the cases where logoColor is not specified.

Reply:
Ah, yeah that makes sense, should I also do the same for the `simple-icons` and store them as their own default colors?

Reply:
Sure, that couldn't hurt!